### ISSUE TITLE
Shipping Labels: Empty state for Package Selected (packages list) screen

### ISSUE THREAD

**Issue by rachelmcr** [woocommerce/woocommerce-ios#4092]
We need some empty state message for the Package Selected screen, when there are no packages set up on the backend.

@koke noted in https://github.com/woocommerce/woocommerce-ios/pull/4073#issuecomment-829892893 that if you don't have any packages set up on your store's backend, the Package Selected screen (which usually shows the packages list) shows a blank screen:

<img src="https://user-images.githubusercontent.com/8739/116658705-ae531f80-a990-11eb-94fc-f910b1887ee8.png" width="300px">

In wp-admin:

![image](https://user-images.githubusercontent.com/8658164/116685741-c041bc00-a9aa-11eb-84cf-eb48182a4d95.png)

**Comment by pmusolino** [comment]
Thanks for discovering this issue @koke and thanks for opening the issue @rachelmcr. I'm moving this to M3 👍

**Comment by rachelmcr** [comment]
I took a closer look at this (preparing to ask for design input) when I realized it might not be needed:

* For M3, we're targeting stores that already have at least one package (see p91TBi-2om-p2):

> Stores with WCS installed, payment method already added, with at least one custom package already created and shipping within the United States.

* For M4, we'll support the custom package creation flow. At that point, the Package Selected screen won't be shown right away if there aren't any packages on the store — instead, selecting the Package Selection field on the Package Details screen will take you into the package creation flow.

As long as we make sure to check for existing packages to determine if a site is eligible for shipping label creation in M3, users won't see this screen in an empty state.

**Comment by rachelmcr** [comment]
I don't think we need an empty state screen here for the M3 release.

How I tested:

1. Started with a test store with WooCommerce Shipping & Tax set up, except no packages added under WooCommerce > Settings > Shipping > WooCommerce Shipping.
2. Set the `shippingLabelsM4` feature flag to `false` (to only test features included in the M3 release).
3. Opened an eligible order on the Orders tab of the app.
4. Confirmed that the "Create Shipping Label" button doesn't appear on the order.
5. Added a package in wp-admin, opened the order again, and confirmed the "Create Shipping Label" button now appears.

There is one case where this doesn't work as expected: If I added packages to my store in wp-admin, and then removed all of them, the `creation_eligibility` endpoint says that my store is still eligible for shipping label creation. In that case, the "Create Shipping Label" appears and I can end up on this blank screen. (This seems to be a bug with the creation eligibility endpoint.) That said, I think this is an unlikely scenario and once we support package creation in the app it won't be an issue anymore.

@pmusolino I'm going to close this, but let me know if you think we should still handle that edge case for M3.
